app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -99,6 +111,33 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+private fun showPhoneNumberSelectionDialog(phoneNumbers: List<String>, onNumberSelected: (String) -> Unit) {
+    val items = phoneNumbers.toTypedArray()
+    val builder = AlertDialog.Builder(this)
+    builder.setTitle(R.string.dialog_title_multiple_phone_numbers)
+
+    val inflater = this.layoutInflater
+    val dialogView = inflater.inflate(R.layout.custom_dialog, null)
```

Comment:
If the variable `inflater` is only used once, can it be inlined where it's called?
```suggestion
    val dialogView = layoutInflater.inflate(R.layout.custom_dialog, null)
```